src/annotator/plugin/pdf.js

```diff
@@ -75,9 +75,32 @@ export default class PDF extends Delegator {
     this._warningBanner = null;
 
     this._checkForSelectableText();
+
+    // Hide annotation layer when the user is making a selection. The annotation
+    // layer appears above the invisible text layer and can interfere with text
+    // selection. See https://github.com/hypothesis/client/issues/1464.
+    this._updateAnnotationLayerVisibility = () => {
+      const selection = /** @type {Selection} */ (window_.getSelection());
+
+      // Add CSS class to indicate whether there is a selection. Annotation
```

Comment:
Again, very essential comment—this would have seemed like magic to a future dev otherwise.